README.md

```diff
@@ -7,7 +7,11 @@
 ![logo_ash_horizontal@4x](https://user-images.githubusercontent.com/25779550/135059025-4cfade5b-bfcb-47e8-872f-8a3d78ce0c25.png)
 
 Serialization toolbox for Akka messages, events and persistent state that helps achieve compile-time guarantee on
-serializability.
+serializability. There're some Akka serialization caveats this tool can help with:
+1. [Missing serialization binding](#missing-serialization-binding)
```

Comment:
Yeah, makes more sense to organize the README around **caveats**, just as in the sphere.it preso